### PR TITLE
Admin edit user page

### PR DESCRIPTION
**Pull request by karenc**
Screenshot

- Move user search column "Sign in as" to right most column

- Add "edit" to actions on admin user search page

- Fix existing admin edit user form so it displays without errors

  - Add app/views/shared/_error_messages.html.erb
  - User can now have multiple email addresses, change @user.email to
    @user.email_addresses
  - Change is_admin field to is_administrator
  - Change @user.is_disabled? to !@user.is_activated?
  - Remove undefined variable "submit_classes"

- Change admin edit user page to use bootstrap styles

- Add verify email button in admin edit user page

- Add controller tests for admin verifying user email addresses

- Remove "Disable Account?" checkbox from admin edit user form

- Add first/last/full names and new email address to admin user edit form

  - Add code to admin users controller #update so it now saves the data
    when the form is submitted
  - Remove "show" and "back" links from admin user edit page, instead
    opens a new browser tab when the user wants to edit a user

- Add user edit tests for admin users controller

- Add password fields to admin user edit form

- Add update password tests for admin users controller

### CODE REVIEW

**Comment by karenc** [comment]
Ready for review.

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -42,5 +45,28 @@ def get_user
...
+      @user.identity.password_confirmation = params[:user][:password_confirmation]
+      return true if @user.identity.save
+      flash[:alert] = "Failed to change password: #{@user.identity.errors.full_messages}"
+      return false
+    end
+
+    def update_user(processed_keys)
+      @user.is_administrator = params[:user][:is_administrator]
+
+      user_params = params[:user].select { |k, v| !processed_keys.include?(k.to_sym) }
```
Comment (jpslav) [inline app/controllers/admin/users_controller.rb]:
```ruby
user_params = params[:user].slice(processed_keys)
```

Comment (karenc) [inline app/controllers/admin/users_controller.rb]: Oh excellent, let me change that.

**Comment by karenc** [comment]
Changes:

```diff
--- a/app/controllers/admin/users_controller.rb
+++ b/app/controllers/admin/users_controller.rb
@@ -10,9 +10,7 @@ module Admin

     def update
       respond_to do |format|
-        processed_keys = [:is_administrator, :email_address, :password, :password_confirmation]
-
-        if change_user_password && add_email_to_user && update_user(processed_keys)
+        if change_user_password && add_email_to_user && update_user
           format.html { redirect_to edit_admin_user_path(@user),
                         notice: 'User profile was successfully updated.' }
         else
@@ -62,10 +60,10 @@ module Admin
       return false
     end

-    def update_user(processed_keys)
+    def update_user
       @user.is_administrator = params[:user][:is_administrator]

-      user_params = params[:user].slice(processed_keys)
+      user_params = params[:user].slice(:first_name, :last_name, :full_name)
       @user.update_attributes(user_params)
     end
   end
```

**Pull request merged**